controllers/kustomization_controller.go

```diff
@@ -150,10 +169,9 @@ func (r *KustomizationReconciler) sync(
 	}
 
 	// apply kustomization
-	cmd = fmt.Sprintf("cd %s && kubectl apply -f %s.yaml", tmpDir, kustomization.GetName())
+	cmd = fmt.Sprintf("cd %s && kubectl apply -f %s.yaml --timeout=5m", tmpDir, kustomization.GetName())
```

Comment:
Do we want this timeout to be configurable in the future? If so, I would add a `// TODO` to keep them visible.

Reply:
Going to use the `interval` since overlapping is not desirable.